src/compiler/utilities.ts

```diff
@@ -2400,7 +2400,7 @@ namespace ts {
      * Serialize an object graph into a JSON string. This is intended only for use on an acyclic graph
      * as the fallback implementation does not check for circular references by default.
      */
-    export const stringify: (value: any) => string = JSON && JSON.stringify

```

Comment:
I'm not sure this was supposed to be part of this PR (though it certainly seems correct!)